### ISSUE TITLE
Bowser UI distorted on iPhone 6S Plus

### ISSUE THREAD

**Issue by tp** [EricssonResearch/bowser#83]
I wanted to try Bowser on an Apple iPhone 6S Plus, but it was not really usable, as the navigation bar is completely missing in portrait, and only partially visible in landscape.

![img_0316](https://cloud.githubusercontent.com/assets/52437/13849927/5b7b6164-ec58-11e5-8711-1538fef2572f.jpg)

**Comment by stefanalund** [comment]
You can pull down the content/web view to show the nav bar.

**Comment by tp** [comment]
Yes, I figured that out on iPad Pro, where it visually works better but then crashes when accessing the mic. Can't send a reproduction there yet since it's an internal app, but will try to dig into that as well.

**Comment by stefanalund** [comment]
@tp could you please submit that as a separate issue? Preferably with a stack trace attached. Thanks.
